### ISSUE TITLE
Simpler / more economic impromptu Root Cellar

### ISSUE THREAD

**Issue by AncientSion** [CleverRaven/Cataclysm-DDA#38143]
![2020-02-19 00_37_28-Window](https://user-images.githubusercontent.com/4209179/74787680-0e8c0300-52b0-11ea-8677-e21624c7b80c.png)

This is the current recipe for a root cellar.

Now, this is pretty extensive in regards to skill requirements and assumes u build a quite a large container.

Now, some impromptu survivalists are building "simpler" root cellars in where they can basically just submerge a barrel into the ground. The deeper you go, the more cold it is (until it isnt).

https://www.youtube.com/watch?v=g_yLLIXh2yI&t=1m31s

Im thinking if the apoc would be hitting, i wouldnt think about building this huge root cellar, but instead rather opt for this simple, less time consuming, "survivor submerged thingy".
Recipe could require perhaps a gallon jug (...) or a 2L Metal tank or something.

Thinking Fab 2, Surv 2, maybe one of them at 3.

https://www.youtube.com/watch?v=tw_lHGbyivM&t=2m30s

Basically, i think the current root cellar is way over the top and most survivors (and preppers actually do it !) would rather use a barrel (wood or metal or plastic all work) and put it into the ground.
Depth 1 to 2 meters.

On youtube you can see that people are actually using these barrel-makeshift root cellars. Way more economic than a full-fledged root cellar especially in a shit hits the fan event.

**Comment by l29ah** [comment]
Yes, especially the current skill requirements are too strict.

**Comment by Amariithynar** [comment]
Both root cellar types can co-exist. In the first video linked, that's a pair of 200L metal barrels sunk into the ground ("requires deep pit") with another, plastic, barrel then placed inside that. The second is just a trash can, which would be about the 100L metal barrel, sunk into a shallow pit. I think for this sort of "simpler" cellar, it should just be a 100L metal barrel, 60L metal tank, and a rope in a shallow pit. Much less storage volume, less cooling ability, but much easier to craft.
